src/main/java/com/deavensoft/paketomat/center/CenterService.java

```diff
@@ -0,0 +1,47 @@
+package com.deavensoft.paketomat.center;
+
+import com.deavensoft.paketomat.center.model.Package;
+import com.deavensoft.paketomat.courier.CourierModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
```

Comment:
Please create CenterService as an interface.
CenterService Impl should implement CenterService.